tests/optimisation_test/optimisation_handler.py

```diff
@@ -0,0 +1,65 @@
+# thread to watch recipe queue
+# thread to watch completed 
+import time
+from archemist.core.persistence.recipe_files_watchdog import RecipeFilesWatchdog
+from archemist.core.state.state import State
+from optimisation_manager import OptimizationState, OptimisationManager
+from recipe_generator import RecipeGenerator
+from pathlib import Path
+from threading import Thread
+import pandas as pd
+
+class OptimizationHandler:
+    def __init__(self, recipe_dir) -> None:
+        self._path_to_config_file = Path.joinpath(Path.cwd(), "tests/optimisation_test/optimization_config.yaml")
+        self._path_to_template_recipe = Path.joinpath(Path.cwd(), "tests/optimisation_test/algae_bot_recipe_template.yaml")
+        self._opt_mgr = OptimisationManager(self._path_to_config_file)
```

Comment:
This should be the other way around the OptimisationManager should construct the OptimisationHandler